openassessment/templates/openassessmentblock/grade/oa_grade_complete.html

```diff
@@ -78,13 +81,15 @@ <h4 class="question__title ui-toggle-visibility__control">
                                 {% endwith %}
                             {% endfor %}
                             {% if grade_details.additional_feedback %}
-                                <li class="question question--feedback ui-toggle-visibility">
-                                    <h4 class="question__title ui-toggle-visibility__control">
-                                        <i class="icon fa fa-caret-right" aria-hidden="true"></i>
-                                        <span class="question__title__copy">{% trans "Additional comments on your response" %}</span>
+                                <li class="question question--feedback is--showing">
```

Comment:
Is "is--showing" still needed here? It was deleted from a lot of other places (in the most recent commit), so just making sure.